src/js/helpers/prepare.js

```diff
@@ -1,11 +1,22 @@
 /* Clearing variables */
 
 import calculateOffset from './calculateOffset';
+import getInlineOption from './getInlineOption';
 
 const prepare = function ($elements, options) {
   $elements.forEach((el, i) => {
+    const positionIn = calculateOffset(el.node, options.offset);
+    const positionOut = positionIn + window.innerHeight;
```

Comment:
Yes, I tested this in a website I’m building, and I encountered the same problem you mentioned.

I guess the solution you proposed it simple and effective.